### ISSUE TITLE
Attempted import error: 'c_background_image_BackgroundImage_xs' is not exported from '@patternfly/react-tokens'.

### ISSUE THREAD

**Issue by idevat** [patternfly/patternfly-react#1121]
After package update I got:

```
Failed to compile.
./node_modules/@patternfly/react-core/dist/esm/components/BackgroundImage/BackgroundImage.js
Attempted import error: 'c_background_image_BackgroundImage_xs' is not exported from '@patternfly/react-tokens'.
```

It is as described in the message:

`BackgroundImage` imports `c_background_image_BackgroundImage_xs`...:
```
$ cat ./node_modules/@patternfly/react-core/dist/esm/components/BackgroundImage/BackgroundImage.js |grep "import.*c_background_image_BackgroundImage_xs"
import { c_background_image_BackgroundImage_lg, c_background_image_BackgroundImage_md, c_background_image_BackgroundImage_md_2x, c_background_image_BackgroundImage_sm, c_background_image_BackgroundImage_sm_2x, c_background_image_BackgroundImage_xl, c_background_image_BackgroundImage_xs, c_background_image_BackgroundImage_xs_2x, c_background_image_Filter } from '@patternfly/react-tokens';
```
...and `c_background_image_BackgroundImage_xs` is not exported from `react-tokens`:
```
$ cat ./node_modules/@patternfly/react-tokens/dist/js/index.js |grep "c_background_image_BackgroundImage"
module.exports.c_background_image_BackgroundImage = {"name":"--pf-c-background-image--BackgroundImage","value":"url(assets/images/pfbg_576.jpg)","var":"var(--pf-c-background-image--BackgroundImage)"}
module.exports.c_background_image_BackgroundImage_2x = {"name":"--pf-c-background-image--BackgroundImage-2x","value":"url(assets/images/pfbg_576@2x.jpg)","var":"var(--pf-c-background-image--BackgroundImage-2x)"}
module.exports.c_background_image_BackgroundImage_sm = {"name":"--pf-c-background-image--BackgroundImage--sm","value":"url(assets/images/pfbg_768.jpg)","var":"var(--pf-c-background-image--BackgroundImage--sm)"}
module.exports.c_background_image_BackgroundImage_sm_2x = {"name":"--pf-c-background-image--BackgroundImage--sm-2x","value":"url(assets/images/pfbg_768@2x.jpg)","var":"var(--pf-c-background-image--BackgroundImage--sm-2x)"}
module.exports.c_background_image_BackgroundImage_lg = {"name":"--pf-c-background-image--BackgroundImage--lg","value":"url(assets/images/pfbg_2000.jpg)","var":"var(--pf-c-background-image--BackgroundImage--lg)"}
```
Package versions:
```
$ cat ./node_modules/@patternfly/react-core/package.json|grep version\":
  "version": "1.43.6"
$ cat ./node_modules/@patternfly/react-tokens/package.json|grep version\":
  "version": "1.9.4"
```
Previously, it worked for me with versions:
```
   "@patternfly/patternfly-next": "^1.0.101",
   "@patternfly/react-core": "^4.0.0",
   "@patternfly/react-tokens": "^1.9.4",
```

**Comment by austincunningham** [comment]
Seeing this issue as well
```
Failed to compile.

./node_modules/@patternfly/react-core/dist/esm/components/BackgroundImage/BackgroundImage.js
Attempted import error: 'c_background_image_BackgroundImage_xs' is not exported from '@patternfly/react-tokens'.
```
On a new react app using first example here https://patternfly-react.surge.sh/patternfly-4/components/page 

dependencies
```js
"dependencies": {
    "@patternfly/react-core": "^1.43.6",
    "react": "^16.7.0",
    "react-dom": "^16.7.0",
    "react-scripts": "2.1.2"
  },
```

**Comment by idevat** [comment]
It works with lower version of react-tokens.

```
$ cat ./node_modules/@patternfly/react-tokens/package.json|grep version\":
  "version": "1.9.1"
$ cat ./node_modules/@patternfly/react-core/package.json|grep version\":
  "version": "1.43.6"
```

**Comment by jschuler** [comment]
Should work in next version, fixed by #1124

**Comment by jschuler** [comment]
Fixed by #1155
